app/src/main/res/layout/history_list_item.xml

```diff
@@ -1,83 +1,87 @@
-            android:layout_width="wrap_content"
-            android:layout_height="match_parent"
-            android:layout_toRightOf="@id/imgHistoryProduct"
-            android:layout_alignLeft="@id/barcodeHistory"
-            android:layout_below="@+id/barcodeHistory"
-            android:layout_toLeftOf="@+id/iconShareHistory"
-            android:textSize="@dimen/font_normal"
-            android:textColor="@color/indigo_400"
-            android:paddingRight="40dp"
-            android:gravity="center_vertical"
-            />
 
-        <TextView
-            android:id="@+id/brandsHistory"
-            android:layout_width="wrap_content"
-            android:layout_height="match_parent"
-            android:layout_toRightOf="@id/imgHistoryProduct"
-            android:layout_alignLeft="@id/titleHistory"
-            android:layout_below="@+id/titleHistory"
-            android:layout_toLeftOf="@+id/iconShareHistory"
-            android:textSize="@dimen/font_normal"
-            android:textColor="@color/indigo_400"
-            android:paddingRight="40dp"
-            android:gravity="center_vertical"
-            />
+    <TextView
+        android:id="@+id/barcodeHistory"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:layout_alignLeft="@id/brandsHistory"
+        android:layout_alignStart="@id/brandsHistory"
+        android:layout_below="@id/brandsHistory"
+        android:layout_toEndOf="@id/imgHistoryProduct"
+        android:layout_toLeftOf="@+id/iconShareHistory"
+        android:layout_toRightOf="@id/imgHistoryProduct"
+        android:layout_toStartOf="@+id/iconShareHistory"
+        android:gravity="center_vertical"
+        android:paddingEnd="40dp"
+        android:paddingRight="40dp"
+        android:paddingStart="40dp"
+        android:textSize="@dimen/font_normal" />
 
-        <ImageButton
-            android:id="@+id/iconShareHistory"
-            android:layout_width="30dp"
-            android:layout_height="30dp"
-            android:scaleType="fitCenter"
-            android:adjustViewBounds="true"
-            android:layout_alignParentRight="true"
-            android:layout_margin="@dimen/spacing_normal"
-            android:layout_marginLeft="12dp"
-            android:layout_marginRight="12dp"
-            android:layout_centerVertical="true"
-            android:background="@color/white"
-            app:srcCompat="@drawable/ic_share_black_48px"/>
+    <ImageButton
+        android:id="@+id/iconShareHistory"
+        android:layout_width="30dp"
+        android:layout_height="30dp"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_centerVertical="true"
+        android:layout_margin="@dimen/spacing_normal"
+        android:layout_marginLeft="12dp"
+        android:layout_marginRight="12dp"
+        android:adjustViewBounds="true"
+        android:background="@color/white"
```

Comment:
Since the card has been removed, the icon background does not match the parent view one. Could you make it transparent? 

I've adjusted the capture B&W levels to emphasize the color mismatch: 
 
![screenshot_1518915637](https://user-images.githubusercontent.com/18316436/36347146-4bf81a3a-1450-11e8-8e26-4b10dbbb3b66.png)

Reply:
@policorn Thanks ! I will fix it.

Reply:
@policorn  Fixed
![fix1](https://user-images.githubusercontent.com/30733262/36347919-36f49a8e-148a-11e8-925d-29f98aa743dd.png)